dynamic_instrumentation/src/point/mod.rs

```diff
@@ -15,6 +15,17 @@ use crate::{arg::InstrumentationArg, hooks::Hooks, util::Convert};
 pub use apply::InstrumentationApplier;
 pub use cast::cast_ptr_to_usize;
 
+/// Sets the priority of this instrumentation. A lower value
+/// signifies higher priority and implies that an instrumentation
+/// with higher priority will be placed before one with lower
+/// priority.
+#[derive(PartialEq, Eq, PartialOrd, Ord, Default, Copy, Clone)]
+pub enum InstrumentationPriority {
+    High,
+    #[default]
+    Low,
+}
+
```

Comment:
The sorting order of the enum as defined is the opposite of what the names suggest.

E.g., the below assertion fails:
```
fn main() {
	#[derive(PartialEq, Eq, PartialOrd, Ord, Default, Copy, Clone)]
	pub enum Priority {
		High,
		#[default]
		Low,
	}

	assert!(Priority::High > Priority::Low)
}
```

Reply:
Yeah, high priority comes first.